templates/web/camden/about/faq-en-gb.html

```diff
@@ -0,0 +1,261 @@
+[% INCLUDE 'header.html', title = loc('Frequently Asked Questions'), bodyclass = 'twothirdswidthpage' %]
+
+[% INCLUDE 'about/_sidebar.html' %]
+
+<h1>Frequently Asked Questions</h1>
+
+<dl>
+
+<dt>What sort of issues can I report?</dt>
+<dd>
+<p>This site is for reporting things which are <strong>broken, dirty,
+damaged</strong> or <strong>dumped</strong>, and need <strong>fixing,
+cleaning</strong> or <strong>clearing</strong>.
+</dd>
+
+<dt>Which issues shouldn't I report?</dt>
+
+<dd>
+<p>Please don’t use this site to inform us of:</p>
+
+<ul>
+    <li>Urgent and emergency problems</li>
+    <li>Complaints about people, including anti-social behaviour</li>
+    <li>Issues with council services, such as bins and recycling</li>
+    <li>Proposals for change, eg to road layouts</li>
+    <li>Complaints about [% c.cobrand.council_name %]</li>
+</ul>
+
+<p><a name="emergencies"></a>For reporting urgent and emergency problems,
+such as a street light with the cover hanging off, or electrical wires where a bollard used to be,
+call 0117 922 2100, Monday to Friday, 8.30am to 6pm, or 0117 922 2050, all other times.
```

Comment:
This is Bristol's FAQ. Unless there's a need for a specific one, the default fixmystreet-uk-council FAQ will be applied, so this file should not be necessary in an initial cobrand.